### DISCUSSION TITLE
당근마켓에서 당근으로, 그 중심에서 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#54] [Announcements]
# 당근마켓에서 당근으로, 그 중심에서 - 정현수 기술 블로그

리브랜딩 랜딩페이지를 만들고 느꼈던 것

https://junghyeonsu.com/posts/daangn-rebranding/

**Comment by Na-hyunwoo** [comment]
중요한 프로젝트의 기회가 주어지셨다는 것이 평소의 잘 쌓아놓은 신뢰자산 덕분이지 않을까요 ?
성공적으로 리브랜딩 하신 것 축하드립니다. 페이지 정말 잘 나온 것 같아요 👍

**Reply by junghyeonsu** [reply]
읽어주셔서 감사합니다. 좋은 말씀도 감사합니다ㅎㅎ
